neptune/new/model_versions_table.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import logging
+from typing import List
+
+from neptune.new.attributes_containers_table import (
+    AttributesContainersTable,
+    AttributesContainersTableEntry,
+)
+from neptune.new.internal.backends.api_model import (
+    AttributeWithProperties,
+)
+from neptune.new.internal.backends.neptune_backend import NeptuneBackend
+from neptune.new.internal.container_type import ContainerType
+
+logger = logging.getLogger(__name__)
+
+
+class ModelVersionsTableEntry(AttributesContainersTableEntry):
+    def __init__(
+        self,
+        backend: NeptuneBackend,
+        id_: str,
```

Comment:
There was `_id` in some of preceding MR-s. Maybe we should always call it fully like `model_id` etc. (or `container_id` or I've missed something)

Reply:
Right, I'll go back to `_id` and we'll discuss potential new name when deciding on id types.

Reply:
Ok, decided that we keep `_id`.